src/module_2/EDA_2.md

```diff
@@ -0,0 +1,1265 @@
+    dtype: int64
+
+
+
+
+```python
+# Cambiar el tipo de datos si es necesario
+# Por ejemplo, si una columna debería ser categórica
+df['alguna_columna'] = df['alguna_columna'].astype('category')
+
+```
+
+
+```python
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# Cargar el DataFrame
+df = pd.read_csv('/home/eduardo_1999/projects/zrive-ds/src/module_2/feature_frame.csv')
+
+# Filtrar solo las columnas numéricas
+numerical_columns = df.select_dtypes(include=['float64', 'int64']).columns
+
+# Configurar el tamaño de la figura
+plt.figure(figsize=(20, 15))
+
+# Crear un boxplot para cada variable numérica
+for i, col in enumerate(numerical_columns):
+    plt.subplot(5, 5, i + 1)  # Cambiar 5, 5 según cuántos subgráficos quieras en cada fila
+    sns.boxplot(x=df[col])
+    plt.title(f'Boxplot de {col}')
+    plt.xlabel(col)  # Etiquetar el eje x con el nombre de la variable
+
+plt.tight_layout()  # Ajustar los espacios entre subgráficos
+plt.show()
+
+
+```
+
+
+    
+![png](EDA_2_files/EDA_2_10_0.png)
+    
+
+
+
+```python
+df.info()
+```
+
+    <class 'pandas.core.frame.DataFrame'>
+    RangeIndex: 2880549 entries, 0 to 2880548
+    Data columns (total 27 columns):
+     #   Column                            Dtype  
+    ---  ------                            -----  
+     0   variant_id                        int64  
+     1   product_type                      object 
+     2   order_id                          int64  
+     3   user_id                           int64  
+     4   created_at                        object 
+     5   order_date                        object 
+     6   user_order_seq                    int64  
+     7   outcome                           float64
+     8   ordered_before                    float64
+     9   abandoned_before                  float64
+     10  active_snoozed                    float64
+     11  set_as_regular                    float64
+     12  normalised_price                  float64
+     13  discount_pct                      float64
+     14  vendor                            object 
+     15  global_popularity                 float64
+     16  count_adults                      float64
+     17  count_children                    float64
+     18  count_babies                      float64
+     19  count_pets                        float64
+     20  people_ex_baby                    float64
+     21  days_since_purchase_variant_id    float64
+     22  avg_days_to_buy_variant_id        float64
+     23  std_days_to_buy_variant_id        float64
+     24  days_since_purchase_product_type  float64
+     25  avg_days_to_buy_product_type      float64
+     26  std_days_to_buy_product_type      float64
+    dtypes: float64(19), int64(4), object(4)
+    memory usage: 593.4+ MB
+
+
+**Analisis**_**de**_**datos**
+
+
+
+
+##Hipótesis 1: El porcentaje de descuento (discount_pct) está positivamente relacionado con la probabilidad de compra (outcome).
```

Comment:
What are the conclusion of this statisical tests?
In addition:

- Are the assumptions (I.I.D) of the tests met?
- You are making multiple hypothesis test, how do you do p-values correction?